pkg/api/k8s_versions.go

```diff
@@ -468,6 +468,7 @@ func getK8sVersionComponents(version string, overrides map[string]string) map[st
 	case "1.11":
 		ret = map[string]string{
 			"hyperkube":                        "hyperkube-amd64:v" + version,
+			"hyperkube-azs":                    "hyperkube-amd64:v" + version + "-azs",
```

Comment:
I thought you were already adding the `-azs` suffix on the version? https://github.com/Azure/aks-engine/commit/39182cdc6579010b74ae8bf6df783a4f5868b9d2#diff-de95d09bcc65a2843f3bad212bdac8e8R80

Reply:
That `version` value comes from `GetAllSupportedKubernetesVersions`. 
Let me know if it is preferred to add a new entry to that map.

Reply:
All good, I think I understand what's going on now.